rgo/git.py

```diff
@@ -149,6 +149,52 @@ def describe(self, pkg=None):
         version = version.replace("-", "_")
         return version, release
 
+    def _check_output(self, cmd):
+        env = dict(os.environ)
+        env["LC_ALL"] = "C.UTF-8"
+        output = subprocess.check_output(cmd, cwd=self.cwd, encoding="utf-8", env=env).strip().splitlines()
+        return output
```

Comment:
Is there really some encoding issue to take care of?

Reply:
TODO: return immediately, don't create the `output` variable